loops.js

```diff
@@ -18,7 +20,14 @@
 // klaar!
  // ==========================================
 
+ const final_number = 4;
 
+ for (let i = 0; i < final_number; i++) {
+        console.log("loop...");
+        if (i === (final_number - 1)) {
```

Comment:
ipv final_number - 1 had je ook i === 4 neer kunnen zetten. Was wellicht iets overzichtelijker geweest, al is deze syntax breder te gebruiken. 

Ik wist niet dat je een if statement kan schrijven zonder een else op z'n minst te noteren. Gaaf!